doc/source/workflows/basics.rst

```diff
@@ -274,10 +282,10 @@ Workflow tasks provide *exactly-once* execution semantics. What this means is th
 
```

Comment:
Suppose f calls g and f has max_retries of 2. Let's assume f calls g and g completes successfully, but f fails after it has received a response from g. Now f will be retried again. Does this mean g will also be retried - even though it completed successfully?

Reply:
@sriram-anyscale  in workflow what you mentioned won't happen because we can't ray.get a `.bind()`.

What you mentioned is:
```
@ray.remote
def g():
   ...

@ray.remote
def  f():
   ray.get(g.bind()) # it's hard to checkpoint in the middle of the python function. We had a prototype of doing this, but it hacks into cpython's code. So we don't have it right now.
   # do something
```